link/src/main/java/com/stripe/android/link/ui/paymentmethod/PaymentMethodViewModel.kt

```diff
@@ -72,16 +72,19 @@ internal class PaymentMethodViewModel @Inject constructor(
     val formController = MutableStateFlow<FormController?>(null)
 
     fun init(loadFromArgs: Boolean) {
+        val cardMap = args.prefilledCardParams?.toParamMap()
+            ?.takeIf { loadFromArgs }
+            ?.let { convertToFormValuesMap(it) }
+            ?: emptyMap()
+        val initialValuesMap = args.initialValueMap
```

Comment:
Should `loadFromArgs` be used here?

Reply:
`loadFromArgs` exists because when you launch Link passing in the pre-filled card params, you'll want the form to be pre-filled with those values. But then if the user navigates within Link and ends up in this screen again, we won't want to load the values from the initial arguments. How does this logic apply to address?